docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -2,18 +2,203 @@
 
 The parent key for all of the following parameters is `openshift4_splunk_forwarder`.
 
-== `namespace`
+
+== `fluentd`
+
+[horizontal]
+type:: dictionary
+default::
++
+[source,yaml]
+----
+fluentd:
+    loglevel: info
+    replicas: 2
+    sharedkey: splunkforwarding
+    ssl:
+      enabled: false
+      cert: ""
+      key: ""
+      passphrase: ""
+----
+
+Fluentd pod parameters.
+
+=== `loglevel`
 
 [horizontal]
 type:: string
-default:: `syn-openshift4-splunk-forwarder`
+default:: "info"
+
+Log level of the splunk-forwarder pod (https://docs.fluentd.org/deployment/logging).
+
+=== `replicas`
+
+[horizontal]
+type:: integer
+default:: 2
 
-The namespace in which to deploy this component.
+Number of replicas of the splunk-forwarder pod.
+
+=== `sharedkey`
+
+[horizontal]
+type:: string
+default:: "splunkforwarding"
+
+The shared key for authentication (see https://docs.fluentd.org/input/forward).
+
+=== `ssl.enabled`
+
+[horizontal]
+type:: boolean
+default:: false
+
+Enable fluentd ssl encryption.
+
+=== `ssl.cert`
+
+[horizontal]
+type:: string
+default:: ""
+
+String of ssl certificate.
+
+=== `ssl.key`
+
+[horizontal]
+type:: string
+default:: ""
+
+String of ssl private key.
+
+=== `ssl.passphrase`
+
+[horizontal]
+type:: string
+default:: ""
+
+Passphrase of ssl private key.
+
+
+== `splunk`
+
+[horizontal]
+type:: dictionary
+default::
++
+[source,yaml]
+----
+splunk:
+  token: 4a8a737d-5452-426c-a6f7-106dca4e813f
+  hostname: splunk.syn-splunk.svc
+  port: 8088
+  insecure: true
+  protocol: https
+  index: openshift
+  sourcetype: openshift
+  source: openshift
+  ca: ""
+----
+
+Parameters for splunk integration.
+
+=== `token`
+
+[horizontal]
+type:: string
+default:: "4a8a737d-5452-426c-a6f7-106dca4e813f"
+
+Splunk HEC token.
```

Comment:
You should document how to get such a token (either here or in the getting started guide)